crit/cmd/cli.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/checkpoint-restore/go-criu/v5/crit"
+	"github.com/spf13/cobra"
```

Comment:
Would you be able to run `make vendor` and include the changes with this commit as well?